app/jobs/post_grouper.rb

```diff
@@ -0,0 +1,19 @@
+#class PostGrouper < ActiveJob::Base
+class PostGrouper
+  def perform(cleanup_gene_claims = true)
```

Comment:
No, because `delete_orphaned_gene_claims` doesn't delete claims from all sources, or only the currently imported source; it specifically deletes ungrouped claims from Ensembl. So in this case the Entrez import happens, grouping happens, and the during post-grouping all claims from Ensembl that we weren't able to group are getting deleted.